pychromepdf/chrome_pdf.py

```diff
@@ -50,7 +48,15 @@ def html_to_pdf(self,html_byte_string, output_file):
             temp_file_path = 'file://{0}'.format(html_file.name)
 
             # prepare the shell command
-            print_to_pdf_command = ' '.join(self._chrome_args).format(
+            args = ['{chrome_exec}'] + self._chrome_options
+
+            if not self._sandbox:
+                # Without sandbox
+                args += ['--no-sandbox']
+
+            args += ['--print-to-pdf={output_file}','{input_file}']
+
+            print_to_pdf_command = ' '.join(args).format(
                 chrome_exec=self._chrome_exe,
                 input_file=temp_file_path,
                 output_file=output_file.name
```

Comment:
Can we move this logic into its own private method? Maybe 
```python
def _generate_shell_command(self, chrome_exe, temp_file_path, output_file_name, sandbox=True):
    .... 
```

Reply:
That’s a good point.
I tried to make the method.